### DISCUSSION TITLE
Adonis Routes

### DISCUSSION THREAD

**Discussion by Hiba-Ed** [adonisjs/core#1576] [Help] [answered]
Hi, I am a beginner and have seen 
```
Route.get('hello', () => 'Hello Adonis')
```
and 
```
Route.get('/hello', () => 'Hello Adonis')
```
being used interchangeably. I have tried the code out myself and both give the same result. Can someone please explain what is the difference between the two? If there is no difference then which one is better to use?

**Comment by thetutlage** [comment, marked as answer]
There's no difference. The forward slash is added automatically, if you don't add one manually
